grab.py

```diff
@@ -1,6 +1,7 @@
 import os
 
-import _grab_ui
+GRAB_DIR = os.path.abspath(os.path.dirname(__file__))
+GRAB_UI_PATH = os.path.join(GRAB_DIR, "_grab_ui.py")
```

Comment:
It’s nice knowing it works for you. It’d also be nice to know if it works because of Kitty 0.21 or because of Python 3.9, because it certainly doesn’t work for me on 0.17.4/3.8.2.

I guess I could put both approaches under a `try`/`except`…